dask_expr/_version.py

```diff
@@ -50,8 +51,8 @@ def get_config() -> VersioneerConfig:
     cfg = VersioneerConfig()
     cfg.VCS = "git"
     cfg.style = "pep440"
-    cfg.tag_prefix = ""
-    cfg.parentdir_prefix = "dask-"
+    cfg.tag_prefix = "v"
```

Comment:
This is the actual relevant change to fix the "unknown" version number for editable installs